_data/sw_clients.yml

```diff
@@ -221,6 +221,7 @@
           sasl-3.1:
           sasl-3.2: 0.13+
           userhost-in-names: 0.13+
+          server-time: Git
```

Comment:
You should add `message-tags`, too, as it's supported and [`server-time` depends on it](https://ircv3.net/specs/extensions/message-tags.html#capabilities ).

Might be worthwhile keeping the list in alphabetical order, too.